tests/test_servefile.py

```diff
@@ -34,6 +51,10 @@ def _run_servefile(args, **kwargs):
             # call servefile as python module
             servefile_path = ['-m', 'servefile']
 
+        # use non-default default port, if one is given via env (and none via args)
+        if '-p' not in args and SERVEFILE_DEFAULT_PORT != 8080:
+            args.extend(['-p', str(SERVEFILE_DEFAULT_PORT)])
```

Comment:
Support for `--port` seems missing and also 8080 is a magic value of "current default port of the actual application. I'd propose to introduce a constant like `DEFAULT_PORT = 8080` over at `servefile.py` and then import and use it here. That would guarantee that these two numbers stay in sync while documenting why it's this precise number.

Reply:
Not adding `--port` here was another "left out intentionally" by me. No test uses `--port` and I don't envision it ever will. Same goes for 8080.... I'd consider that domain knowledge and wouldn't just for this import a constant from servefile.

If we end up going with a different default port for the tests than 8080 this constant will be taken out though.

Reply:
Magic port value gone, `--port` added.